crates/subspace-farmer/src/plot.rs

```diff
@@ -411,6 +411,20 @@ impl WeakPlot {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
I'd implement `Ord` explicitly here since we care about very specific order over one field only, just to be clear.

Reply:
Idk though. Standard library uses derive everywhere, for instance for `Result` and `Option`. It is kinda handy to find just derive and look at the order of declaration of fields.

Reply:
My thought process here is that in many cases we care about "some" defined order, while here we care about very specific order. The code works correctly as is of course.